### PR TITLE
config.jelly: add "Copy HTML reports in workspace" option

### PR DESCRIPTION
**Pull request by bagage**

### CODE REVIEW

File: bootstraped-multi-test-results-report/src/main/java/com/github/bogdanlivadariu/jenkins/reporting/cucumber/CucumberTestReportPublisher.java [hunk 1]
```diff
@@ -147,8 +154,20 @@ public boolean perform(AbstractBuild build, Launcher launcher, BuildListener lis
                 if (featuresResult) {
                     result = Result.SUCCESS;
                 } else {
-                    result = isMarkAsUnstable() ? Result.UNSTABLE : Result.FAILURE;
+                    result = markAsUnstable ? Result.UNSTABLE : Result.FAILURE;
                 }
+
+                // finally copy to workspace, if needed
+                if (copyHTMLInWorkspace) {
```
Comment (BogdanLivadariu) [inline bootstraped-multi-test-results-report/src/main/java/com/github/bogdanlivadariu/jenkins/reporting/cucumber/CucumberTestReportPublisher.java]: use the getter `isCopyHTMLInWorkspace`

File: bootstraped-multi-test-results-report/src/main/java/com/github/bogdanlivadariu/jenkins/reporting/cucumber/CucumberTestReportPublisher.java [hunk 2]
```diff
@@ -147,8 +154,20 @@ public boolean perform(AbstractBuild build, Launcher launcher, BuildListener lis
                 if (featuresResult) {
                     result = Result.SUCCESS;
                 } else {
-                    result = isMarkAsUnstable() ? Result.UNSTABLE : Result.FAILURE;
+                    result = markAsUnstable ? Result.UNSTABLE : Result.FAILURE;
```
Comment (BogdanLivadariu) [inline bootstraped-multi-test-results-report/src/main/java/com/github/bogdanlivadariu/jenkins/reporting/cucumber/CucumberTestReportPublisher.java]: use getter `isMarkAsUnstable` same goes for other occurrences as well

**Comment by BogdanLivadariu** [comment]
@bagage two minor things, other than than, this is good to go

**Comment by bagage** [comment]
Done, thanks @BogdanLivadariu

**Comment by BogdanLivadariu** [comment]
:+1:  great job, I just hope you have tested it :smile:

**Comment by bagage** [comment]
Yes I tested it on my side, working great for me ;).

**Comment by BogdanLivadariu** [comment]
@bagage super, will try to hit the deploy, in the next days, working on a mechanism to make travis deploy the releases when a tag is created, I hope this will not take too long

**Comment by bagage** [comment]
There is no urgency in deploying for me - I am using my own built `.hpi` so please take your time ;).

**Pull request merged**